src/components/ClusterOverview/TopConsumers/ClusterTopConsumers.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import { TopConsumers } from './TopConsumers';
+import { InlineLoading } from '../../Loading';
+import { ClusterOverviewContextGenericConsumer } from '../ClusterOverviewContext';
+import { PodModel, NodeModel } from '../../../models';
+import { formatBytes as utilFormatBytes } from '../../../utils';
+
+const formatCpu = (cpuTime, fixed = 0) => Number(cpuTime).toFixed(fixed);
+
+const formatBytes = memory => {
+  const bytes = utilFormatBytes(memory);
+  return `${bytes.value} ${bytes.unit}`;
+};
+
+const getConsumers = (kind, results, nameLabel, formatLabel) => {
```

Comment:
maybe this would better fit to selectors

Reply:
moved to `selectors/prometheus/consumers`